auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -258,6 +291,7 @@ interface Authenticator {
      *
      * @return A [VimeoRequest] object to cancel API requests.
      */
+    @Deprecated("Deprecated in favor of checkSsoConnection")
```

Comment:
Because the approach still technically works, and it is a better practice to deprecate functionality before removing it in order to give any consumers that were using it some leeway.